templates/admin/payments/workshop/change_form.html

```diff
@@ -0,0 +1,6 @@
+{% extends "admin/change_form.html" %}
+
+{% block extrahead %}
+    {{ block.super }}
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Bootstrap is stomping on some of the admin styles (buttons that are too short, chopping off text) and seems like it might not be necessary.